docs/src/examples/components/Button/Types/ButtonExample.shorthand.steps.ts

```diff
@@ -0,0 +1,3 @@
+const config: ScreenerTestsConfig = { themes: ['teams', 'teamsDark', 'teamsHighContrast'] }
```

Comment:
Example of adding `themes` prop without `steps`; theme switching should still work